interfaces/builtin/locale_control.go

```diff
@@ -70,6 +72,7 @@ func init() {
 		name:                  "locale-control",
 		summary:               localeControlSummary,
 		implicitOnClassic:     true,
+		implicitOnCore:        release.OnCoreDesktop,
```

Comment:
Nitpick, I don't know what is the design behind this but I would much rather have:

```
implicitOnClassic: true
implicitOnCore: false
implicitOnCoreDesktop: true
```

and not have to import release, think about mocking and anything like that.

CC @pedronis on design.

What we do now is deviate from the existing approach where core and classic are distinc entities and something else cares about release. If we follow through logical consequence we could just have `implicit` with a boolean expression checking values of the release package, but I think this is less desirable, IMO.

Reply:
We have used `implicitOnCore: release.OnCoreDesktop` before here:

https://github.com/snapcore/snapd/blob/ea13a335211bd7e249cc353df75db0524c4edc0b/interfaces/builtin/steam_support.go#L406-L409

For this particular one, we're relying on some support in our boot base snap for this interface to work. Namely:

1. Make `/etc/locale.conf` and `/etc/vconsole.conf` symlinks to `/etc/writable`.
2. Another systemd patch to let systemd-localed update these files: https://launchpadlibrarian.net/735989212/systemd_255.4-1ubuntu8_255.4-1ubuntu8.1+ucd1.diff.gz

It's something that could work on regular Ubuntu Core, with the equivalent changes. Not sure whether we'd want to make the implcit slot available before that happens though.

Reply:
this is the design we agreed on for the time being, ideally there shouldn't be more/many implicit slots on Core Desktop than on Core, everything should ideally com from snaps that are not base. Making implicitCoreDesktop first class would negate that. builtin is importing release to look at OnClassic here and there anyway